content/md/first-steps/manage-your-attributes.md

```diff
@@ -268,31 +268,97 @@ Unlike products you cannot restore or revert an attribute’s version.
 
 # Delete an attribute
 
+There are two ways to remove an attribute:
+* Through the attribute edit form
+* Through the delete shortcut in the grid
+
+You can also choose if you want to loose or to keep the data used by this attribute.
+
+::: warning
+If you cannot access the attribute page or if you cannot add/delete an attribute, it is probably because you do not have the permission to.  
+Please refer to [Manage your user roles](build-your-user-roles.html) to learn about roles.
+:::
+
+::: warning
+Please first delete your attribute from the families it is used in and then from the products. After this, you will be able to delete safely your attribute.
```

Comment:
honest question, why do you need to remove it from the families first ?

Reply:
Don't know, this sentence was already written. I deleted it.